lib/WP_Auth0_LoginManager.php

```diff
@@ -462,10 +467,11 @@ public function login_user( $userinfo, $id_token, $access_token ) {
 	 * @param bool   $is_new - `true` if the user was created in the WordPress database, `false` if not.
 	 * @param string $id_token - user's ID token returned from Auth0.
 	 * @param string $access_token - user's access token returned from Auth0; not provided when using implicit_login().
+	 * @param string $refresh_token - user's refresh token returned from Auth0; not provided when using implicit_login().
 	 *
 	 * @throws WP_Auth0_BeforeLoginException - Errors encountered during the auth0_before_login action.
 	 */
-	private function do_login( $user, $userinfo, $is_new, $id_token, $access_token ) {
+	private function do_login( $user, $userinfo, $is_new, $id_token, $access_token, $refresh_token ) {
```

Comment:
what's the difference between `login_user` and `do_login` methods?

Reply:
`login_user` is called after auth is successful so it decides if someone can login or if we need to make an account. `do_login` is called if `login_user` is successful, handles core WP login and hooks.

Reply:
can these get renamed to something more clear? 

i.e.:

 `login_user` is processing an authentication and deciding what to do next. Shouldn't that be something like `process_successful_auth`?

and `do_login` well, it's more straight forward. But it could be `do_wp_login`